classes/fields/number.php

```diff
@@ -250,11 +249,9 @@ public function validate( $value, $name = null, $options = null, $fields = null,
 		$dot         = $format_args['dot'];
 
 		$check = str_replace(
-			array( $thousands, $dot, html_entity_decode( $thousands ) ), array(
-				'',
-				'.',
-				'',
-			), $value
+			array( $thousands, $dot, html_entity_decode( $thousands ) ),
+			array( '', '.', '', ),
```

Comment:
* :no_entry_sign: Comma not allowed after last value in single-line array declaration
* :no_entry_sign: Comma not allowed after last value in single-line array declaration

<sub>hash: 360ed25b00088f5e0370af2acdb62f07</sub>